Rubberduck.Parsing/Symbols/IdentifierReference.cs

```diff
@@ -132,6 +133,28 @@ public bool IsSelected(QualifiedSelection selection)
                    Selection.ContainsFirstCharacter(selection.Selection);
         }
 
+        public virtual (string context, Selection highlight) HighligthSelection(ICodeModule module)
```

Comment:
I do not really like that this method is here. It solely deals with presentation concerns. This should rather be a method somewhere in `Core.UI`.

Reply:
Good idea, moved it to `Rubberduck.UI.Controls.IdentifierReferenceExtensions` (same namespace as the `SearchResultsViewModel`).